cfme/containers/provider/openshift.py

```diff
@@ -98,6 +98,12 @@ def num_template_ui(self):
     def from_config(prov_config, prov_key, appliance=None):
         token_creds = OpenshiftProvider.process_credential_yaml_key(
             prov_config['credentials'], cred_type='token')
+        hawkular_hostname = prov_config['endpoints']['hawkular'].hostname
```

Comment:
You just need to make these a little more robust, at the moment if the hawkular endpoint is missing, this will fall over